spec/till_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'till'
+
+describe Till do
+  subject(:till) {described_class.new(read_json_klass, file_klass, json_klass, filename, calculate_total_klass, receipt_printer_klass)}
```

Comment:
Metrics/LineLength: Line is too long. [135/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)